src/Altinn.App.PlatformServices/Extensions/ConfigurationBuilderExtensions.cs

```diff
@@ -15,11 +16,21 @@ public static class ConfigurationBuilderExtensions
         /// <param name="args">The original command line arguments</param>
         public static void LoadAppConfig(this IConfigurationBuilder builder, string[] args = null)
         {
-            builder.AddJsonFile(
-                new PhysicalFileProvider("/"),
-                @"altinn-appsettings-secret/altinn-appsettings-secret.json",
-                true,
-                true);
+            try
+            {
+                builder.AddJsonFile(
+                    new PhysicalFileProvider("/altinn-appsettings-secret"),
+                    @"altinn-appsettings-secret.json",
+                    true,
+                    true);
+            }
+            catch (DirectoryNotFoundException)
+            {
+                // Extra secrets configuration file is optional. The directory usually does not exist, but might
```

Comment:
Some nitpicking: The directory will "always" be there. The only exception is in a dev environment.

Reply:
Good to know! Thanks. Updated the comment.